### PR TITLE
Host is taken into account to calculate the record id.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: tests/test_changes.py [hunk 1]
```diff
@@ -30,6 +30,15 @@ def test_a_change_record_is_updated_per_bucket_collection(self):
         self.assertNotEqual(change_record['last_modified'],
                             after['last_modified'])
 
+    def test_collection_on_different_host_do_not_have_the_same_id(self):
+        self.app.post_json(self.records_uri, SAMPLE_RECORD,
```
Comment (Natim) [inline tests/test_changes.py]: @leplatrem I suppose you may prefer:

```
diff --git a/tests/test_changes.py b/tests/test_changes.py
index f2157cf..9623913 100644
--- a/tests/test_changes.py
+++ b/tests/test_changes.py
@@ -31,13 +31,14 @@ class UpdateChangesTest(BaseWebTest, unittest.TestCase):
                             after['last_modified'])
 
     def test_collection_on_different_host_do_not_have_the_same_id(self):
-        self.app.post_json(self.records_uri, SAMPLE_RECORD,
-                           headers=self.headers)
+        # A POST already happend in setUp
+        resp = self.app.get(self.changes_uri, headers=self.headers)
+        previous_count = len(resp.json['data'])
         self.app.app.registry.settings['http_host'] = 'another.host.com'
         self.app.post_json(self.records_uri, SAMPLE_RECORD,
                            headers=self.headers)
         resp = self.app.get(self.changes_uri, headers=self.headers)
-        self.assertEquals(len(resp.json['data']), 2)
+        self.assertEquals(len(resp.json['data']), previous_count + 1)
 
     def test_changes_bucket_and_collection_are_created_automatically(self):
         self.app.delete('/buckets/monitor', headers=self.headers)
```

Comment (leplatrem) [inline tests/test_changes.py]: that's alright :)

File: tests/test_changes.py [hunk 2]
```diff
@@ -30,6 +30,15 @@ def test_a_change_record_is_updated_per_bucket_collection(self):
...
+                           headers=self.headers)
+        self.app.app.registry.settings['http_host'] = 'another.host.com'
```
Comment (leplatrem) [inline tests/test_changes.py]: no sideeffect when the next tests are executed ?

**Comment by leplatrem** [comment]
r+wc

File: tests/test_changes.py (hunk 2 above)
Comment (Natim) [inline tests/test_changes.py]: No because the setup will apparently recreate the app.

**Pull request merged**